deployment/build_and_release/modules/release/variables.tf

```diff
@@ -14,8 +15,25 @@ variable "bucket_env" {
 
 variable "signing_keyring_location" {
   description = "The GCP location to create the signing keyring"
+  type        = string
 }
 
 variable "tf_state_location" {
   description = "The GCP location to store Terraform remote state"
+  type        = string
+}
+
+variable "cloudbuild_path" {
+  description = "The path of the Cloud Build config in the Github repo"
+  type        = string
+}
+
+variable "cloudbuild_tag" {
+  description = "TODO"
```

Comment:
Can we have a description here. Suggestions:
 - put trigger in the variable name to make it clear that this controls how it is triggered
 - describe that an empty string does no trigger on tag
 - (optional) add a default of empty string, so that only the triggers that need to be specified can be

Same for the branch below.

Reply:
I added a link to the documentation and put in a default value.